recipe/meta.yaml

```diff
@@ -123,3 +123,4 @@ extra:
     - kenodegard
     - mbargull
     - ocefpaf
+    - beeankha
```

Comment:
@beeankha you agree to be a maintainer here?

Reply:
Yes, I agree 👍